binary_transparency/firmware/api/map.go

```diff
@@ -28,3 +30,28 @@ type DeviceReleaseLog struct {
 	DeviceID  string
 	Revisions []uint64
 }
+
+// MapCheckpoint is a commitment to a map built from the FW Log at a given size.
+// The map checkpoint contains the checkpoint of the log this was built from, with
+// the number of entries consumed from that input log. This allows clients to check
+// they are seeing the same version of the log as the map was built from. This also
+// provides information to allow verifiers of the map to confirm correct construction.
+type MapCheckpoint struct {
+	LogCheckpoint LogCheckpoint
+	LogSize       uint64
+	RootHash      []byte
+	Revision      uint64
+}
+
+// MapInclusionProof contains the value at the requested key and the proof to the
+// requested tree size.
```

Comment:
To the requested tree size or to the map root?

Reply:
Checkpoint, good catch.